src/qcodes/instrument_drivers/cryomagnetics/Cryomagnetics4G.py

```diff
@@ -0,0 +1,381 @@
+from __future__ import annotations
+
+import logging
+import re
+import time
+from dataclasses import dataclass
+
+from qcodes.instrument import VisaInstrument
+from qcodes.validators import Enum, Numbers
+
+log = logging.getLogger(__name__)
```

Comment:
Is this log used anywhere? Seems like the logging adaptor attached to the instrument is used

Reply:
Thanks, missed removing this!